Makefile

```diff
@@ -92,6 +92,12 @@ benchmarks-stateful-storage:
 benchmarks-overhead:
 	./scripts/run_benchmark.sh -p overhead
 
+benchmarks-capacity:
+	./scripts/run_benchmark.sh -p capacity
+
+benchmarks-frequency-tx-payment:
+	./scripts/run_benchmark.sh -p frequency-tx-payment
+
```

Comment:
This will need a re-sync from `main`; significant changes to the way benchmarking is invoked.